workflow.txt

```diff
@@ -1,10 +1,9 @@
+fork on github
 git clone
-git checkout -b my-solution
-git commit -m ""
 git fetch --all
+git checkout master
 git add .
-git pull upstream master
+git commit -m ""
 git merge my-solution
```

Comment:
Close, the typical flow for submitting hw via a pull request is :
```
fork on github
git clone
git checkout -b my-solution
git add .
git commit -m ""
git push origin my-solution
```
then you can make a pull request from github